lib/modules/manager/npm/extract/index.spec.ts

```diff
@@ -113,6 +114,33 @@ describe('modules/manager/npm/extract/index', () => {
       });
     });
 
+    it('returns an array of dependencies when is a yaml version too', async () => {
```

Comment:
Unclear description

Reply:
the test is the same as previous test, and has a similar text, I changed it a little bit, but since is the same test for the different format, I think it should have the same (or similar description)